src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -287,8 +287,9 @@ class CompanyStep extends React.Component {
                     <View style={styles.mt4}>
                         <StatePicker
                             label={this.props.translate('companyStep.incorporationState')}
-                            onChange={value => this.clearErrorAndSetValue('incorporationState', value)}
+                            onInputChange={value => this.clearErrorAndSetValue('incorporationState', value)}
                             value={this.state.incorporationState}
+                            errorText={this.getErrorText('incorporationState')}
                             hasError={this.getErrors().incorporationState}
```

Comment:
Shouldn't we remove these?

Reply:
Thanks for pointing out these changes, Yes I need to change all these places where we are using StatePicker,

https://github.com/Expensify/App/blob/ad637d3dd7dbb03cfdc6ebcb9196944c4f6165bb/src/pages/ReimbursementAccount/AddressForm.js#L75

https://github.com/Expensify/App/blob/ad637d3dd7dbb03cfdc6ebcb9196944c4f6165bb/src/pages/ReimbursementAccount/CompanyStep.js#L292